strapi/strapi
# [DEITS] add results table to import

### What does it do?

- Adds the progress table to import (which can't be seen until import is actually implemented)
- `engine.transfer()` now returns progress results along with source and provider results

### Why is it needed?

So that import has a pretty table of results

### How to test it?

Finish developing (or mock) the import process and then run `yarn strapi import --file filename` and see the results table displayed on successful import.
